README.md

```diff
@@ -1,10 +1,10 @@
+Themis is an innovative iPad app providing a comprehensive and optimized way for tutors to assess submitted programming exercises on the go. As a complementary extension to the [Artemis](https://github.com/ls1intum/Artemis) platform, an open-source learning management system used at universities, Themis allows tutors to effortlessly inspect students’ submissions and provide feedback.
 
-Designed as a native iPad application, Themis optimizes viewing and assessing programming exercises on a mobile device. Its intuitive interface and user-friendly assessment editor enable tutors to give both general as well as code-specific inline feedback. Leveraging the iPad’s unique features allows a more convenient and streamlined feedback process using Themis. By supporting the Apple Pencil, tutors can highlight specific code segments and provide comments using the Scribble feature.
+Designed as a native iPad application, Themis optimizes viewing and assessing exercises on a mobile device. Its intuitive interface and user-friendly assessment editor enable tutors to give general as well as referenced feedback for programming, text, modeling, and file upload exercises. Leveraging the iPad’s unique features allows a more convenient and streamlined feedback process using Themis. By supporting the Apple Pencil, tutors can highlight specific segments and provide comments using the Scribble feature.
 
-Through the connection to the Artemis server, tutors can log in and select the corresponding course as usual, to seamlessly continue the assessment on the go. To enhance the feedback process even further, Themis assists tutors while providing more suitable and effective feedback. For this, [Themis-ML](https://github.com/ls1intum/Themis-ML) is incorporated, a system for automated feedback suggestions of programming exercises. By analyzing the code and comparing past feedback from previous assessments, fitting feedback to current submissions is proposed. This smart feedback is displayed for tutors to review and modify as needed, improving the accuracy and efficiency of the assessment process.
+Through the connection to the Artemis server, tutors can log in and select the corresponding course as usual to seamlessly continue the assessment on the go. To enhance the feedback process even further, Themis assists tutors by providing feedback suggestions. By analyzing submissions and comparing past feedback from previous assessments, feedback suggestions fitting to current submissions are proposed. The suggestions are displayed for tutors to review and modify as needed, improving the accuracy and efficiency of the assessment process.
 
 ## Support
 In case of any problems, please open a GitHub issue or send an email to the [Themis Developer Team](mailto:ios2223cit.dse@xcit.tum.de).
```

Comment:
Not sure if this email is still in use 🤔

Reply:
Don't know either, I just removed it 🙂